tests/render/test_renderer_configuration.py

```diff
@@ -91,6 +91,10 @@ def test_successful_renderer_configuration_instantiation(
 
 
 @pytest.mark.unit
+@pytest.mark.xfail(
+    reason="This test will fail until RendererConfiguration._validate_configuration_or_result is re-enabled.",
+    strict=True,
+)
```

Comment:
Could we put the version we `x-failed` this? We have a history of forgetting to resolve `x-fails` (I'm very much a culprit here) - do we have a strategy to ensure this will be tackled? Perhaps added to that ticket?

Reply:
I will add the version and link this in the JIRA ticket.